.circleci/config.yml

```diff
@@ -307,6 +323,8 @@ jobs:
           keys:
             - *cache_key_win
       - run: yarn install --frozen-lockfile --cache-folder ../.cache/yarn
+      #TEMP: test
```

Comment:
Do we still need this line?  I imagine so if we are wanting to build windows via bazel here as well

Reply:
I'm calling bazelisk/bazel directly to work around a strange issue that the angular project also encountered. At the moment I'm just pushing a bunch of tests to try to get the CI working for windows tests.